web-services-app/src/main/java/edu/unc/lib/boxc/web/services/processing/DownloadImageService.java

```diff
@@ -0,0 +1,67 @@
+
+package edu.unc.lib.boxc.web.services.processing;
+
+import edu.unc.lib.boxc.model.api.DatastreamType;
+import edu.unc.lib.boxc.search.api.models.ContentObjectRecord;
+import org.springframework.core.io.InputStreamResource;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URL;
+import java.util.Objects;
+
+/**
+ * @author snluong
+ */
+public class DownloadImageService {
+    private String iiifBasePath;
+
+    public ResponseEntity<InputStreamResource> streamImage(String pidString, String size)
+            throws IOException {
+
+        String url = buildURL(pidString, size);
+        InputStream input = new URL(url).openStream();
+        InputStreamResource resource = new InputStreamResource(input);
+
+        return ResponseEntity.ok()
+                .header(HttpHeaders.CONTENT_DISPOSITION, "attachment; filename=image_" + size + ".jpg")
+                .contentType(MediaType.IMAGE_JPEG)
+                .body(resource);
+    }
+
+    /*
+    A method that builds the IIIF URL based on an assumption of full region, 0 rotation, and default quality.
+     */
+    private String buildURL(String id, String size) {
+        var formattedSize = size;
+        if (!Objects.equals(size, "full")) {
```

Comment:
It'd be good to define a constant for `"full"` since it's used in a few places, and it needs to be the same between the service and the controller.